api/v1beta1/foundationdbcluster_types.go

```diff
@@ -2127,3 +2150,26 @@ const (
 	// PublicIPSourceService specifies that a pod gets its IP from a service.
 	PublicIPSourceService PublicIPSource = "service"
 )
+
+// ProcessClass models the different FDB classes
+type ProcessClass string
```

Comment:
Where is this type being used?

Reply:
I use it in some places like: https://github.com/FoundationDB/fdb-kubernetes-operator/pull/380/files#diff-9e300e65cd8198dd2dcc3e8f7a252ecea4fa5d072e3bf55bb1aff38aa4927d7fR222 + https://github.com/FoundationDB/fdb-kubernetes-operator/pull/380/files#diff-9e300e65cd8198dd2dcc3e8f7a252ecea4fa5d072e3bf55bb1aff38aa4927d7fR242 + https://github.com/FoundationDB/fdb-kubernetes-operator/pull/380/files#diff-9e300e65cd8198dd2dcc3e8f7a252ecea4fa5d072e3bf55bb1aff38aa4927d7fR538

Reply:
I initially thought it makes sense to have a general `ProcessClass` type but I think we actually don't need it and I will remove it.